docs/src/@primer/gatsby-theme-doctocat/components/requires-js-flash.js

```diff
@@ -1,11 +1,13 @@
 import React from 'react'
-import {Flash, Link} from '@primer/components'
+import {Link} from 'gatsby'
+import Note from '@primer/gatsby-theme-doctocat/src/components/note'
 
 function RequiresJSFlash() {
   return (
-    <Flash mb={3}>
-      This component requires JavaScript to function. Please refer to the <Link href="/#installation">Installation section</Link> to set it up.
-    </Flash>
+    <Note>
+      This component requires JavaScript to function. Please refer to the{' '}
```

Comment:
the `{' '}` feels weird to me 🤔

Reply:
it's a Prettier thing when wrapping text, the alternative is doing `the {}` or keeping it in one line, but I think it's better to go with what Prettier says 😄